docs/builtin.rst

```diff
@@ -182,6 +182,14 @@ Available
      - Calculate (f)ALFF and aggregate using spheres placed on coordinates
      - Done
      - 0.0.1
+   * - :class:`junifer.markers.EdgeCentricFCParcels`
+     - Calculate edge-centric functional connectivity over parcellation
```

Comment:
@LeSasse Is there any related publication explaining the method?

Reply:
It's there in the class' docstrings.

Reply:
Like you mean in the `builtin.rst`?

Reply:
we can also add this one as it was published a bit earlier (by the same group): https://www.nature.com/articles/s41593-020-00719-y

Reply:
> Like you mean in the `builtin.rst`?

yes

The idea is that people looking at markers list can know what they are.

Reply:
this paper is also on topic but by a different group and more of a reaction to the eFC method, but may be an interesting read for users: https://www.nature.com/articles/s41467-022-29775-7

Reply:
> > Like you mean in the `builtin.rst`?
> 
> yes
> 
> The idea is that people looking at markers list can know what they are.

Which column should it go in then?

Reply:
I have added the reference.